diff --git a/node_modules/detox/src/artifacts/utils/temporaryPath.js b/node_modules/detox/src/artifacts/utils/temporaryPath.js
--- a/node_modules/detox/src/artifacts/utils/temporaryPath.js
+++ b/node_modules/detox/src/artifacts/utils/temporaryPath.js
@@ -12 +12,4 @@
-const globAsync = promisify(glob);
+const globAsync =
+  typeof glob === 'function'
+    ? promisify(glob)
+    : (pattern, options) => glob.glob(pattern, options);
diff --git a/node_modules/detox/src/logger/utils/streams/DetoxJSONLParser.js b/node_modules/detox/src/logger/utils/streams/DetoxJSONLParser.js
--- a/node_modules/detox/src/logger/utils/streams/DetoxJSONLParser.js
+++ b/node_modules/detox/src/logger/utils/streams/DetoxJSONLParser.js
@@ -1 +1 @@
-const JsonlParser = require('stream-json/jsonl/Parser');
+const JsonlParser = require('stream-json/jsonl/parser.js').default;
@@ -13 +13 @@
-    const writable = JsonlParser.make({ checkErrors: true })
+    const writable = JsonlParser.asStream({ checkErrors: true })
